library/Icingadb/Widget/ObjectsStatistics.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/* Icinga DB Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget;
+
+use Icinga\Chart\Donut;
+
+use Icinga\Module\Icingadb\Widget\Detail\ObjectStatistics;
+use ipl\Html\Text;
+use ipl\Html\ValidHtml;
+use ipl\Html\HtmlString;
+
+/**
+ * Objects statistics
+ */
+class ObjectsStatistics extends ObjectStatistics
+{
+    protected $summary;
```

Comment:
Please add phpDoc.

Reply:
Sure, but don't forget to add the `DependencyNodeSummary` as a possible type for this property in #1069.